libs/dh-app/core/feature-shell/src/lib/shell/shell.component.html

```diff
@@ -0,0 +1,12 @@
+<watt-shell>
+  <ng-container sidenav>
+    <p>Sidenav</p>
+  </ng-container>
+
+  <ng-container toolbar>
+    <p>Toolbar</p>
+  </ng-container>
+
+  <h1>THIS IS THE APP</h1>
+  <watt-autocomplete></watt-autocomplete>
+</watt-shell>
```

Comment:
suggestion: add router outlet for app features

```suggestion
  <router-outlet></router-outlet>
</watt-shell>
```